build_table.py

```diff
@@ -57,15 +58,15 @@ def read_npm(self, path, github_name):
                     self.data[github_name][job]['version'] = version
 
     def read_zuul(self, info, github_name):
-#        if github_name != 'mediawiki-extensions-Echo':
-#            return
+        #if github_name != 'mediawiki-extensions-Echo':
```

Comment:
We should kill this block entirely, I don't remember what it's for, probably some debugging thing :/